src/commands/choose-persona.ts

```diff
@@ -0,0 +1,42 @@
+import { APIApplicationCommandOptionChoice, SlashCommandBuilder } from "discord.js"
+
+import { BotCommand } from '../types'
+import { Persona } from "../data/persona"
+import { db } from "../data/database"
+import { forcePotdChange } from "../helpers/potd-helpers"
+
+if (0 === db.persona.getCount()) {
+    await db.persona.load()
```

Comment:
We force-load the personas if there is none in the DB, this is used when deploying commands